_posts/2023-02-28-admin-panels-for-index-operations.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title:  "Index management UI enhancements"
+authors:
+- ihailong
+- gbinlong
+- suzhou
+- lxuesong
+date: 2023-02-28
+categories:
+ - technical-post
+meta_keywords: index management, index mappings, index operation UI, shrink index operations, OpenSearch Dashboards
+meta_description: Simplify cluster operations with index management UI enhancements that enable you to open, close, reindex, shrink, and split indexes in a more user-friendly way.
+
+excerpt: OpenSearch now provides users with a new interface to execute common indexing and data stream operations. You can now perform create, read, update, delete (CRUD) and mapping operations for indexes, index templates, and aliases through the OpenSearch Dashboards UI. You can also open, close, reindex, shrink, and split indexes. This step marks the beginning of creating an Index Management UI, which will serve as a unified administration panel in OpenSearch Dashboards. 
+---
+
+Managing indexes, aliases, and templates in OpenSearch Dashboards can be difficult for users using only the API. There are restrictions and relation-binding tasks, such as adding aliases to indexes, simulating an index template by its name, and more. We are pleased to announce that these cluster operations have been significantly simplified with the Index Management UI enhancements in OpenSearch Dashboards v2.5.
+
+### Visual editor for index mappings with nested tree editor
+
+Building a JSON for index mappings is complicated because it has several nested layers and properties. To simplify this task, we now provide a visual editor, which is shown in the following image, with editable nested properties. You can add properties by clicking the operation buttons and then see what the mappings will look like by switching to the JSON editor.
+
+<img src="/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/index-mapping-editor.png" alt="index mapping editor"/>{: .img-fluid }
+
+### Simulate index by index name
+
+It is challenging to predict the appearance of the index taking into account existing templates. To resolve this issue, we will attempt to match the index name with the templates whenever you change the index name. We will merge what users manually input with what the matching template contains, making it a "what you see is what you get" index.
+
+<img src="/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/simulate-template.png" alt="simulate template"/>{: .img-fluid }
+
+### Edit settings by JSON editor and diff mode
+
+The new Index Management UI offers a visual editor and a JSON editor, which you can use if the visual editor does not support all available fields. Additionally, we provided the editor with a `diff` mode, as shown in the following image, so that you can compare the existing index and see what changes they have made to ensure that no mistakes are made.
```

Comment:
How about "so that you can compare the existing index with the index in old version"

Reply:
That works, but you need a "the" before "old": "so that you can compare the existing index with the index in the old version." Made the change in the suggestion.